Makefile

```diff
@@ -205,7 +205,19 @@ test-e2e: e2e-setup
 			--up-local \
 			--go-test-flags "-timeout=15m" \
 			--local-operator-flags "$(ZAP_FLAGS)" \
-			$(OPERATOR_SDK_EXTRA_ARGS)
+			$(OPERATOR_SDK_EXTRA_ARGS) \
+			| tee test-e2e.log
+	$(Q)cat test-e2e.log | grep 'Local operator stderr:' | sed -e 's,.*Local operator stderr: \(.*\)}\\n",\1,g' \
+		| sed -e 's,\\a,\a,g' \
```

Comment:
A side comment on `sed` usage, you can concatenate expressions in a single call using `;`. For instance:

```bash
sed -e 's,\\a,\a,g;s,\\b,\b,g'
```

Reply:
Ah, thanks for the tip ;)